app/validators/notation.js

```diff
@@ -0,0 +1,13 @@
+import _ from "npm:lodash"
+
+export default function validateNotation() {
+  return (key, oldValue) => {
+    let checkStandardNotation = _.filter(Object.values(oldValue), standard => {
+      return !_.isEmpty(standard.statementNotation)
+    })
+
+    let isValid = checkStandardNotation.length / Object.values(oldValue).length
+
+    return isValid >= 0.3 ? true : "notation needed"
```

Comment:
The naming of isValid suggests it's a boolean (which I think is good), but here, it's a decimal. I'd move the comparison to where you're defining isValid. Also, add a comment above that line as it's not entirely clear what's going on there either (I mean, I get it, but had we not talked, I'd be more confused)